typescript/react/best-practice/react-props-in-state.jsx

```diff
@@ -1,6 +1,6 @@
 class Test1 extends React.Component {
-// TODO: react-props-in-state
   constructor() {
+    // todo:react-props-in-state
```

Comment:
Do you want [`todook` or `todoruleid`](https://semgrep.dev/docs/writing-rules/testing-rules/) here?

Reply:
yep, thanks, that is `todoruleid`
waiting for https://github.com/returntocorp/semgrep/issues/2369 to be solved